app/stac_api/urls.py

```diff
@@ -45,4 +46,5 @@
         AssetDetail.as_view(),
         name='asset-detail'
     ),
+    path(f'{HEALTHCHECK_ENDPOINT}', CollectionList.as_view(), name='health-check')
```

Comment:
This won't work because only path behing `/api` are accessible from outside so healthcheck should be `{API_BASE}/{HEALTHCHECK_ENDPOINT}`

Reply:
Thanks - I didn't know that